lang/strings_english.txt

```diff
@@ -1718,3 +1718,5 @@ $MANTIS_ERROR[ERROR_DEPRECATED_SUPERSEDED] = 'Deprecated functionality: "%1$s",
 $MANTIS_ERROR[ERROR_DISPLAY_USER_ERROR_INLINE] = 'Warning: The system is configured to display MantisBT errors (E_USER_ERROR) inline. Program execution will continue; this may lead to system/data integrity issues.';
 $MANTIS_ERROR[ERROR_TYPE_MISMATCH] = 'Data Type mismatch. Enable detailed error messages for further information.';
 $MANTIS_ERROR[ERROR_BUG_CONFLICTING_EDIT] = 'This issue has been updated by another user, please return to the issue and submit your changes again.';
+$MANTIS_ERROR[ERROR_SPAM_SUSPECTED] = 'Exceeded allowed limit, blocked to avoid spam.';
```

Comment:
Suggest rewording to make the error message more user friendly.

```
You have reached the allowed activity limit of %d events within the last %d minutes; your action has been blocked to avoid spam, please try again later.
```

I believe it would be even better if we indicated "please try again in xxxx minutes", but calculating that would either require another new API function to get the date of the user's earliest change within the spam window, or renaming  history_count_recent_events() to history_get_user_recent_events() and make it return an array.